radix-engine/src/state_manager/deprecated_staging.rs

```diff
@@ -1,185 +0,0 @@
-use crate::ledger::*;
```

Comment:
_Note from author:_

This file was using `HashMap`s, and I figured it is easier to delete the deprecated thing than migrate it.
I myself am 100% sure that this staging implementation is no longer usable with state manager (so even keeping it as some "reference impl" for test purposes seems no longer appropriate).